### DISCUSSION TITLE
Add util nodes to menu other than Functions?

### DISCUSSION THREAD

**Discussion by aVersionOfReality** [bnpr/Malt#143] [Q&A] [answered]
I am adding some of my own utility nodes. The default is that they all appear as new entries under the Functions menu when adding a node. This menu is already large. As I add to it, it is becoming difficult to navigate. I'd rather add my new nodes under a new entry in the main Add Node menu.

Is it possible to add new nodes to the other menu sections when using the Shader Library path?

I assume if I drop new nodes into Malt's files themselves then they will go in the specified area, but then they get uninstalled whenever I update Malt.

**Comment by pragma37** [comment, marked as answer]
All functions will be under the functions menu regardless of where they are declared.
You can distribute them under different entries in the functions menu if they are in different GLSL files.

I personally add nodes using the F3 search menu for the most part.
